<launch>
  <rosparam file="$(find rm_base)/test/test_imu.yaml" command="load"/>

  <!-- load the controllers -->
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
        output="screen" args="
        controllers/imu_sensor_controller
"/>

  <node name="rm_base" pkg="rm_base" type="rm_base" respawn="false"/>

</launch>